app/misc/rodauth_main.rb

```diff
@@ -360,14 +360,6 @@ class RodauthMain < Rodauth::Rails::Auth
   end
   # rubocop:enable Layout/LineLength
 
-  def admin?
-    member && member.admin? # rubocop:disable Style/SafeNavigation
-  end
-
-  def member?
-    member && member.member? # rubocop:disable Style/SafeNavigation
-  end
```

Comment:
removed `admin?` and `member?` because we have only these two access levels for Member